internal/controller/standalone_pgadmin/pod.go

```diff
@@ -242,28 +242,52 @@ func podConfigFiles(configmap *corev1.ConfigMap, pgadmin v1beta1.PGAdmin) []core
 }
 
 func startupScript(pgadmin *v1beta1.PGAdmin) []string {
-	// loadServerCommand is a python command leveraging the pgadmin setup.py script
+	// loadServerCommandv7 is a python command leveraging the pgadmin v7 setup.py script
```

Comment:
:pick: For this and the v8 variable, I'd prefer camel case, i.e.
```suggestion
	// loadServerCommandV7 is a python command leveraging the pgadmin v7 setup.py script
```